packages/client-core/src/admin/DefaultAdminRoutes.tsx

```diff
@@ -23,84 +23,130 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
   },
-  allowedRoutes.routes && {
+  routes: {
     name: 'user:dashboard.routes',
-    path: '/admin/routes',
+    scope: 'routes',
+    component: RoutesComp,
+    access: false,
     icon: <Icon type="Shuffle" style={{ color: 'white' }} />
   },
-  allowedRoutes.location && {
+  locations: {
     name: 'user:dashboard.locations',
-    path: '/admin/locations',
+    scope: 'location',
+    component: Locations,
+    access: false,
     icon: <Icon type="NearMe" style={{ color: 'white' }} />
   },
-  allowedRoutes.instance && {
+  instance: {
     name: 'user:dashboard.instances',
-    path: '/admin/instance',
+    scope: 'instance',
+    component: Instance,
+    access: false,
     icon: <Icon type="DirectionsRun" style={{ color: 'white' }} />
   },
-  allowedRoutes.channel && {
+  avatars: {
+    name: 'user:dashboard.avatars',
+    scope: 'globalAvatars',
+    component: Avatars,
+    access: false,
+    icon: <Icon type="Accessibility" style={{ color: 'white' }} />
+  },
+  benchmarking: {
+    name: 'user:dashboard.benchmarking',
+    scope: 'benchmarking',
+    component: Benchmarking,
+    access: false,
+    icon: <Icon type="Timeline" style={{ color: 'white' }} />
+  },
+  bots: {
+    name: 'user:dashboard.bots',
+    scope: 'bot',
+    component: BotSetting,
+    access: false,
+    icon: <Icon type="SmartToy" style={{ color: 'white' }} />
+  },
+  channel: {
     name: 'user:dashboard.channels',
-    path: '/admin/channel',
+    scope: 'channel',
+    component: Channels,
+    access: false,
     icon: <Icon type="CalendarViewDay" style={{ color: 'white' }} />
   },
-  allowedRoutes.user && {
-    name: 'user:dashboard.users',
-    path: '/admin/users',
-    icon: <Icon type="SupervisorAccount" style={{ color: 'white' }} />
-  },
-  allowedRoutes.invite && {
+  invites: {
     name: 'user:dashboard.invites',
-    path: '/admin/invites',
+    scope: 'invite',
+    component: Invites,
+    access: false,
     icon: <Icon type="PersonAdd" style={{ color: 'white' }} />
   },
-  allowedRoutes.globalAvatars && {
-    name: 'user:dashboard.avatars',
-    path: '/admin/avatars',
-    icon: <Icon type="Accessibility" style={{ color: 'white' }} />
+  recordings: {
+    name: 'user:dashboard.recordings',
+    scope: 'recording',
+    component: Recordings,
+    access: false,
+    icon: <Icon type="Videocam" style={{ color: 'white' }} />
   },
-  allowedRoutes.static_resource && {
+  resources: {
     name: 'user:dashboard.resources',
-    path: '/admin/resources',
+    scope: 'static_resource',
+    component: Resources,
+    access: false,
     icon: <Icon type="PermMedia" style={{ color: 'white' }} />
   },
-  allowedRoutes.benchmarking && {
-    name: 'user:dashboard.benchmarking',
-    path: '/admin/benchmarking',
-    icon: <Icon type="Timeline" style={{ color: 'white' }} />
-  },
-  allowedRoutes.settings && {
+  settings: {
     name: 'user:dashboard.setting',
-    path: '/admin/settings',
+    scope: 'settings',
```

Comment:
Is there a way we can control subsettings in admin panel? Like a user can only manage client-settings and not the other settings.

Reply:
hmm, we will need separate scopes for each setting type in that case anyway, so we should drive it from that